webTest/js/script.js

```diff
@@ -0,0 +1,70 @@
+// function hideShow(){
+//
+//   if(cheker%2=0){
+//     var hider=document.querySelector("#hide");
+//     hider.style.display="none";
+//   }
+// }
+var cheker = 2;
+var clikeShowHide = document.querySelector("#showHide");
+var hider = document.querySelector(".showOne");
+clikeShowHide.addEventListener('click', function() {
+    if (cheker % 2 === 0) {
+        hider.style.display = hider.style.display + " none";
+        cheker++;
+    } else {
+        hider.style.display = hider.style.display.replace("none", "");
+        cheker++;
+    }
+});
+
+var prBtn = document.querySelector('button');
+var contactName = document.querySelector('#contName');
+var contactEmail = document.querySelector('#conEmail');
+var contactPhone = document.querySelector('#conPhone');
+
+prBtn.addEventListener('click', function() {
+
+    if (contactName.value.length === 0) {
```

Comment:
you could also check `if (!contactName.value)` .. try that and see if it works - same for other checks